taskcluster_util/taskcluster_download.py

```diff
@@ -102,6 +102,15 @@ def _check_crendentials_file(self, options):
             logger.info('Load Credentials from {}'.format(abs_credentials_path))
         except Exception as e:
             logger.warning('No credentials. Run with "--help" for more information.')
+            if options.credentials:
```

Comment:
This value is always defined [because of this line](https://github.com/askeing/taskcluster-util-python/blob/fix_19/taskcluster_util/taskcluster_download.py#L55). Hence, if you don't provide any credential file, you'll end up with:
```
WARNING: No credentials. Run with "--help" for more information.
ERROR: Please check your credentials file: /home/jlorenzo/tc_credentials.json
ERROR: [Errno 2] No such file or directory: '/home/jlorenzo/tc_credentials.json'
```